compute/cker/src/train/MaxPool.test.cc

```diff
@@ -225,6 +232,50 @@ TEST(CKer_Operation, MaxPool2D)
     std::vector<float> expected_input_deriv = {0, 0, 0.2, 0};
     verifier.verifyBackward(output_deriv, expected_input_deriv);
   }
+
+  // with min, max params
```

Comment:
(Optional) Using the specific activation name looks good to me.
```suggestion
  // with ReLU6
```